pkg/kubectl/cmd/create_role.go

```diff
@@ -267,6 +267,19 @@ func (c *CreateRoleOptions) validateResource() error {
 			return err
 		}
 	}
+
+	if len(c.ResourceNames) > 0 {
+		illegalVerbs := []string{}
+		for _, verb := range c.Verbs {
+			switch verb {
+			case "list", "watch", "create", "deletecollection":
```

Comment:
Can you comment why only these 4 are allowed?
I think others are also OK, like "*" or "patch".